blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -96,6 +96,17 @@ public String getStringProperty(String name) {
         return props.getProperty(name);
     }
 
+    public static String getRawStringProperty(Enum<? extends ConfigDefaults> name) {
+        return getRawStringProperty(name.toString());
+    }
+    public static String getRawStringProperty(String name) {
+        return props.getProperty(name);
+    }
+
+    public static boolean containsKey(String key) {
```

Comment:
Technically, no. They could also be instance methods. If they're static, they allow two modes of access:
1. `Configuration.getInstance().getRawStringProperty(...)`
2. `Configuration.getRawStringProperty(...)`

If they were instead instance methods, only the first way would work. Honestly, it's mostly just a matter of taste at this point. I eventually want to re-work the class so that the only  static methods are factory methods. Before then, these "raw" methods would go away. They are currently there just so I can read from the `props` field without having to make it public, and without invoking the wierd logic in the normal `getStringProperty` method. I want to refactor the `getStringProperty`, but need more tests across the whole code base before I do that.